bundle-minimal/pom.xml

```diff
@@ -53,4 +53,18 @@
 			<artifactId>log4j-web</artifactId>
 		</dependency>
 	</dependencies>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-war-plugin</artifactId>
+					<configuration>
+						<failOnMissingWebXml>false</failOnMissingWebXml>
+					</configuration>
+				</plugin>
```

Comment:
Is this just to configure the property or to create the war output file? Because this minimal bundle exists to run as a monolithic application, I didn't expect this here.

Reply:
It's in pluginManagement, thus to set this as default for other projects that inherit from the bundle-minimal pom.

Reply:
Ah yes, I missed the 'pluginManagement' tag. Clear.